src/components/Slide/slide.css

```
@@ -1,3 +1,48 @@
+.slide__viewport {
+  --slide-mobile-viewport-width: 100dvw;
+  position: relative;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)